index.js

```diff
@@ -165,7 +178,7 @@ class S3Adapter {
         if (data && !data.Body) {
           return reject(data);
         }
-        return resolve(data.Body);
```

Comment:
Here I cast `data.Body` to `string` which is most probably wrong as we should pass up a stream?